lib/unsupported-features/node-builtins-modules/process.js

```diff
@@ -17,6 +17,10 @@ const process = {
     execArgv: { [READ]: { supported: ["0.7.7"] } },
     execPath: { [READ]: { supported: ["0.1.100"] } },
     exitCode: { [READ]: { supported: ["0.11.8"] } },
+    features: {
```

Comment:
I notice I have missed a load of `features` :face_exhaling: 
![image](https://github.com/user-attachments/assets/1e45cb9d-1709-447a-a4df-d3fb3716af1f)

Reply:
These were indeed missed from your issue, but were already included in older versions, hence why I didn't include them here

I'm happy to include them in this PR as well _or_ in a follow-up PR, whatever you guys prefer is fine for me

Reply:
This'll need to be a follow up! I'll make an issue.

Reply:
Issue here #360